backend-server/src/api/controllers/availabilityController.ts

```diff
@@ -54,9 +54,11 @@ export async function replaceAllAvailabilities(
     body.organization
   );
 
-  if (!availabilities.length) {
-    throw new Error("Can not construct availabilities from given input");
-  }
+  // // I think we want to allow zero length availabilities 
```

Comment:
@adirar111 I commented this out because I was getting 500 errors when I tried to save an empty calendar